optimum/habana/transformers/models/llama/modeling_llama.py

```diff
@@ -428,6 +431,17 @@ def __init__(self, config: LlamaConfig, layer_idx: Optional[int] = None):
             self.v_proj = None
         self.inp_seq_len = -1
         self.norm_factor = 1.0 / math.sqrt(self.head_dim)
+        self.fused_scaled_dot_product_attention = (
+            ModuleFusedSDPA(
+                FusedSDPA,
+                scale=self.norm_factor,
+                attention_dropout=self.attention_dropout,
+                enable_recompute=False,
+                flash_attention_fp8=getattr(config, "flash_attention_fp8", False),
```

Comment:
Why not
```suggestion
                flash_attention_fp8=config.flash_attention_fp8,
```
?

Reply:
Because [Harish Subramony](https://github.com/hsubramony) noticed that in some cases this field is not set, so `config.flash_attention_fp8` would fail with an AttributeError

Reply:
@hsubramony Can you give the commands or describe the cases where this error happens please?

Reply:
tests/transformers/tests/models/llama/test_modeling_llama.py::LlamaModelTest::test_eager_matches_sdpa_generate
tests/test_trainer.py::[1mGaudiTrainerIntegrationTest::test_multiple_peft_adapters

- AttributeError: 'LlamaConfig' object has no attribute 'flash_attention_fp8'